help/include/configure-topic-notifications.md

```diff
@@ -0,0 +1,22 @@
+{start_tabs}
+
+{tab|desktop-web}
+
+{!topic-actions.md!}
+
+1. Configure topic notifications using the row of icons at the top of the menu.
+
+!!! tip ""
+
+    You can also configure notifications by clicking the topic notifications icon
```

Comment:
We might want to use the term "topic status" instead since this would be consistent with the **Status** column in **Topic settings**.

```suggestion
    You can also configure notifications by clicking the topic status icon
```

Reply:
What do you think about something like "notifications status"? It seems like "status" could potentially refer to resolved/unresolved.

Reply:
Yes, I think "notifications status" would work better.